client/asset/bch/bch.go

```diff
@@ -212,15 +212,15 @@ func NewWallet(cfg *asset.WalletConfig, logger dex.Logger, network dex.Network)
 		// Bitcoin Cash don't take a change_type argument in their options
 		// unlike Bitcoin Core.
 		OmitAddressType: true,
-		// Bitcoin Cash uses estimatefee instead of estimatesmartfee, and even
-		// then, they modified it from the old Bitcoin Core estimatefee by
-		// removing the confirmation target argument.
-		FeeEstimator: estimateFee,
-		AssetID:      BipID,
+		AssetID:         BipID,
 	}
 
 	switch cfg.Type {
 	case walletTypeRPC, walletTypeLegacy:
+		// Bitcoin Cash uses estimatefee instead of estimatesmartfee, and even
+		// then, they modified it from the old Bitcoin Core estimatefee by
+		// removing the confirmation target argument.
+		cloneCFG.FeeEstimator = estimateFee
```

Comment:
Nice catch moving this under here.  Looks like `LegacyBalance:      cfg.Type != walletTypeSPV,` above would be best done in this `case` as well, but it's not wrong presently, so nbd.